.woodpecker.yml

```diff
@@ -4,24 +4,15 @@
 variables:
   - &muslrust_image "clux/muslrust:1.67.0"
 
-# Broken for cron jobs currently, see
-# https://github.com/woodpecker-ci/woodpecker/issues/1716
-# clone:
-#   git:
-#     image: woodpeckerci/plugin-git
-#     settings:
-#       recursive: true
-#       submodule_update_remote: true
+clone:
+  git:
+    image: woodpeckerci/plugin-git
+    settings:
```

Comment:
https://hub.docker.com/r/woodpeckerci/plugin-git

Seems fine to me. Not sure what it could be.